package.json

```
@@ -1,13 +1,13 @@
 {
   "name": "akamai-edgeworkers-cli",
-  "version": "1.8.1",
+  "version": "1.9.0",
   "description": "A tool that makes it easier to manage Akamai EdgeWorkers code bundles and EdgeKV databases. Call the EdgeWorkers and EdgeKV API from the command line.",
   "repository": "https://github.com/akamai/cli-edgeworkers",
   "scripts": {
     "build": "npm run build-ts",
     "clean": "rm -rf node_modules",
     "build-ts": "tsc",
-    "postinstall": "if test \"$NODE_ENV\" != \"production\" ; then npm run build ; fi",
+    "postinstall": "node scripts/postinstall",
```

We might need to change this to
```suggestion
    "postinstall": "node scripts/postinstall.js",
```

---

@kyleli-akamai , please update in our internal mirror.

---

Sure.